api/docs/sandbox.md

```diff
@@ -0,0 +1,132 @@
+# ONDC Mock & Sandbox
+
+ONDC [Mock & Sandbox](https://mock.ondc.org/sandbox/services) features an ExpressJS Web Server, a Vite ReactJS App. This README specifies the instructions on how to use it.
+
+# Mock Server
+This service will imitate various API behaviors like search, on_search, select, etc. You may use this server to expedite the implementation of your APIs.
+- If you are a buyer app (BAP), you can provide /action APIs payload and you will receive the subsequent sync and async responses.
+You'll either get an ACK or NACK as a sync response based on schema validations performed on your provided payload.
+If you receive a NACK as sync response, it indicates there are schema errors in your payload that you need to address.
+
+<div style="text-align:center">
+<img src="https://github.com/abhinavv245/b2b_mock_server/blob/draft-mock_server/docs/images/Sync_response_nack.png?raw=true" alt="Asyn response" width="300" height="300">
+</div>
+
+- If you receive an ACK sync response, you'll then get the subsequent async response. For example, if you provided a payload for the /action API, you'll receive an async response for the subsequent /on_action API. A notable feature of this service is the ability to select from various scenarios like location not serviceable, item out of stock, etc.
+
+<div style="text-align:center">
+<img src="https://github.com/abhinavv245/b2b_mock_server/blob/draft-mock_server/docs/images/async_response.png?raw=true" alt="Asyn response" width="300" height="300">
+</div>
+
+- If you are a seller app (BPP), you can provide /on_action APIs payload and you will receive the subsequent sync and async request. Once you receive ACK as a sync response, you will receive the subsequent async request. For e.g. if you have provided the payload for /on_action API, then you will receive an async request for next /action API. 
+
+# Sandbox
+
+- In this sandbox environment, you can freely test your APIs. However, to receive successful responses, it's necessary to be subscribed to the staging registry. This server will require and authenticate the authorization header for verification. Once you are subscribed to the staging registry and have developed the APIs, you can test them by providing the payload and authorization header. You may refer to this [document](https://github.com/ONDC-Official/developer-docs/blob/main/registry/signing-verification.md) and use the [utilities](https://github.com/ONDC-Official/reference-implementations/tree/main/utilities/signing_and_verification) to create the authorization header. 
```

Comment:
authenticate the authorization header - validate the authorization header.

auth header can also be generated and validated using the signcheck navigation. However, it is suggested to use this only for testing.